superset/connectors/druid/models.py

```diff
@@ -1188,7 +1189,12 @@ def run_query(  # druid
         )
 
         # the dimensions list with dimensionSpecs expanded
-        dimensions = self.get_dimensions(groupby, columns_dict)
+
+        if IS_SIP_38:
```

Comment:
This could be condensed to, 

```python
dimensions = self.get_dimensions(columns if IS_SIP_38 else groupby, columns_dict)
```

Reply:
Thanks @john-bodley , I'll update the PR with these readability improvements and merge once CI passes.